similar.info

```diff
@@ -1,4 +1,4 @@
 type = module
 name = Similar Entries
-description = Provides similar content view based on relevancy scores.
+description = Uses relevancy scores to provide a list of published content that is similar to the content being viewed.
```

Comment:
> published content

Hm...  Content won't necessarily be published -- it's a view so it is user-configurable. 

> similar to the content being viewed.

Actually, it's similar to whatever node's nid is passed into the view as a contextual filter, not necessarily "content being viewed".

I think this description is a little too specific.

Reply:
What about something like...

`Uses relevancy scores to determine similarity between different pieces of content. A View is provided.`